src/components/DatePicker/DatePicker.tsx

```diff
@@ -49,7 +59,7 @@ const DatesPicker: React.FC<DatesPickerProps> = ({formik, ...props}: DatesPicker
                                     ...params.InputProps,
                                     className: homePageStyles.inputDateField
                                 }}
-                                label={props.fieldName}
+                                label={selectLabel(props.fieldName)}
```

Comment:
Maybe we can just use `props.label` passed from HomePage, it already has the values you're getting from your inner function.

Reply:
Resolved! :)